server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
The idea make sense. However it would be breaking for us until we re-factor our consumption of it.

I'm  planning to do a front end refactor of hobknob (Reactify) within the next few month so thats when we would start working on refactoring our deployment also. 

I'd probably keep this on hold for the moment :)

Also @tombuildsstuff we should probably setup son kind of forum to share what both your plans are for hobknob so we don't get us doing the same stuff!

\\cc @matteofigus @tombuildsstuff